<launch>   

  <node name="kimm_kortex_custum_home" pkg="kimm_kortex_custum" args="-keyboard" type="kimm_kortex_custum_home" output="screen">
  <param name="port_number" type="int" value="10000"/>
  </node>
</launch>
